pandora-client-web/src/components/directMessage/directMessage.tsx

```diff
@@ -94,8 +96,17 @@ function DirectChannelInput(): ReactElement {
 		}
 	});
 
+	React.useEffect(() => {
+		ref.current?.focus();
+	}, [channel.account]);
```

Comment:
[non-blocking] We might want to create a hook similar to what we have in `chatInput.tsx` "Handler to autofocus chat input": Hook that listens to global keypresses and focuses the input if no other input is focused.